content/how-to/add-code-servers/github-cloud/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Adding a GitHub Cloud code server"
-description: "How to add a GitHub cloud code server to your tenant"
-lead: "How to add a GitHub Cloud code server to your tenant"
```

Comment:
Actually, this was a good lead and description. They're different from the title and they accurately describe the page. I wouldn't remove these.